OpenIPSL/Electrical/Renewables/PSSE/InverterInterface/BaseClasses/LVACM.mo

```diff
@@ -0,0 +1,46 @@
+within OpenIPSL.Electrical.Renewables.PSSE.InverterInterface.BaseClasses;
+model LVACM "Low Voltage Active Control Management"
+        parameter Real lvpnt0 "Low voltage point for low voltage active current management (pu)";
```

Comment:
If it is PerUnit then use the proper type and remove (pu) from the description string. Also fix this for the other occurences.